src/components/Session/Deposit.jsx

```diff
@@ -21,18 +21,22 @@ export default class Deposit extends React.Component {
     let currencyPicker;
     let anchors;
     if (allAssets.length == 0) {
-      const noTrustLines = (<div>
+      const noTrustLines = (<div className="Generic__content">
```

Comment:
Capitalized classes are supposed to be scoped to just a single .jsx file. So the class `Generic_*` is just for `Generic.jsx`.

Please either copy and paste the styles or use Generic

Reply:
ok, will copy it over to the Deposit css file